### PR TITLE
**Fix:** form and form elements

### PR DESCRIPTION
**Pull request by TejasQ**
# Summary
This PR:

- Fixes `LabelText` to have a correct line-height
- Improves `Switch` to layout options if specified
- Fixes the outdated `README` of `Select`
- Fixes incorrect layout handling on `Input`
- Fixes the layout of `Form` groups
- Fixes the `z-index` and layout of `Autocomplete`

Tester 1

- [ ] Things look good on the demo.

### CODE REVIEW

**Comment by fabien0102** [comment]
Looks nice, can you publish a canary version so I can try in my app 😃

File: src/Form/Form.tsx [hunk 1]
```diff
@@ -14,6 +14,9 @@ const Container = styled("form")(({ theme }) => ({
   "> :not(:last-child) > *": {
     marginBottom: theme.space.small,
   },
+  "> * > *:not(:last-child)": {
```
Comment (stereobooster) [inline src/Form/Form.tsx]: Space between children _inside_ groups  _inside_ groups?

File: src/Form/README.md [hunk 2]
```diff
@@ -41,3 +80,88 @@ const { name, gitUrl, branch, isCreating, isValid } = {
...
+  // You can even debounce this.
+  setState({ search: text })
+  if (text.length) {
+    setState({ loading: true })
+    fetch("https://dog.ceo/api/breeds/list")
+      .then(results => results.json())
+      .then(results =>
+        setState({
+          data: results.message
+            .filter(name => ~name.indexOf(text))
```
Comment (stereobooster) [inline src/Form/README.md]: `name.includes(text)`?

File: src/utils/constants.ts [hunk 3]
```diff
@@ -136,6 +136,7 @@ const font = {
     main: `"Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif`,
     code: "Menlo, monospace",
   },
+  /** 1.4 */
```
Comment (stereobooster) [inline src/utils/constants.ts]: ?

Comment (micha-f) [reply src/utils/constants.ts]: Shows up in the editor in hover :)

Comment (TejasQ) [reply src/utils/constants.ts]: For this

image

**Pull request merged**